src/js/main.js

```diff
@@ -306,3 +306,56 @@ if (document.body.dataset.page === "run") {
       notebookPreviewEl.appendChild(div);
     });
 }
+
+// === Integrations Page ===
+
```

Comment:
You can wrap this code in `if (document.body.dataset.page === "run") {` similarly to other pages above, this way it doesn't run on homepage, etc, which would cause errors (due to missing elements).